### ISSUE TITLE
Overlap on x axis makes text unreadable

### ISSUE THREAD

**Issue by jethror1** [eastgenomics/trendyQC#108]
To reproduce:
- plot CEN assay and plot gc_percent on y axis

<img width="1678" alt="image" src="https://github.com/eastgenomics/trendyQC/assets/45037268/e841bbbb-00a8-4f3e-b234-1ad3b2081192">

**Comment by jethror1** [comment]
- myeloid and pct coverage on x axis
<img width="1532" alt="image" src="https://github.com/eastgenomics/trendyQC/assets/45037268/c980dde7-af53-4590-89ec-e5cb6bbb7020">

**Comment by Yu-jinKim** [comment]
Yeah i had noticed that but i don't really have a solution for this. Unless plotly has a nice solution for this, i don't see an easy solution

**Comment by mattgarner** [comment]
How about removing that text? The month/year is already present within the runfolder names, so I don't think it's essential to keep both
